diff -rup Swift-3.3.3-php5/lib/Swift.php Swift-3.3.3-php5-split/lib/Swift.php
--- Swift-3.3.3-php5/lib/Swift.php	2007-10-14 02:37:51.000000000 -0400
+++ Swift-3.3.3-php5-split/lib/Swift.php	2012-12-05 17:48:04.857969611 -0500
@@ -274,7 +274,7 @@ class Swift
     for ($i = 1, $len = count($list); $i < $len; $i++)
     {
       $extension = substr($list[$i], 4);
-      $attributes = split("[ =]", $extension);
+      $attributes = explode("[ =]", $extension);
       $this->connection->setExtension($attributes[0], (isset($attributes[1]) ? array_slice($attributes, 1) : array()));
     }
   }
